### PR TITLE
Update imide.json

### PR DESCRIPTION
**Pull request by imide**
add discord verification, email via improvmx.

### CODE REVIEW

**Comment by imide** [comment]
i fixed the json

i hate json oml

**Review by YouFoundAlpha — CHANGES_REQUESTED** [review]
Please add an email, it is required and for Discord verification please read https://www.is-a.dev/docs/discord_verification/

File: domains/imide.json [hunk 1]
```diff
@@ -5,6 +5,12 @@
         "discord":  "555915168932954112"
     },
     "record": {
-        "CNAME": "imide-is-a-dev.pages.dev"
+        "CNAME": "imide-is-a-dev.pages.dev",
```
Comment (Stef-00012) [inline domains/imide.json:8]: You can't use CNAME with other records

also for discord, you need to make a `_discord.imide` domain

**Review by Stef-DP — APPROVED** [review]

**Review by Stef-DP — DISMISSED** [review]
approved for error before

File: domains/imide.json (hunk 1 above)
Comment (imide) [reply domains/imide.json:8]: the docs do not provide any info regarding mail. what is the standard/recommended way to handle that?

**Comment by imide** [comment]
> Please add an email, it is required and for Discord verification please read https://www.is-a.dev/docs/discord_verification/

my email was removed as it was my personal email with my full name. another reviewer said I don't need the email if I have the discord id in there instead. I'm also making my mail so I don't need to expose my full name and personal email

**Comment by CuteDog5695** [comment]
> Please add an email, it is required and for Discord verification please read https://www.is-a.dev/docs/discord_verification/

no? as long as you put any other socials in the file (like discord, twitter and etc.) it should be fine

**Comment by CuteDog5695** [comment]
oh btw issue still needs to be fixed

**Comment by imide** [comment]
> oh btw issue still needs to be fixed

the docs don't say anything regarding custom emails. any tips/recommendations?

**Review by iostpa — APPROVED** [review]
seems fine to me now

**Comment by CuteDog5695** [comment]
> > oh btw issue still needs to be fixed
> 
> the docs don't say anything regarding custom emails. any tips/recommendations?

like making a custom email with is-a.dev domain?

**Comment by CuteDog5695** [comment]
> > > > oh btw issue still needs to be fixed
> > > 
> > > 
> > > the docs don't say anything regarding custom emails. any tips/recommendations?
> > 
> > 
> > like making a custom email with is-a.dev domain?
> 
> yeah

free way you can do is either use a email forwarding service (like ImprovMX which is free), but you can also consider using Zoho Mail which is free (tho you cant really use it with gmail, you have to use their email dashboard to send emails i think)

**Comment by CuteDog5695** [comment]
oh yea, with CNAME you cant use MX and TXT so with cloudflare pages you are out of luck

**Pull request merged**